src/test/java/de/retest/recheck/cli/subcommands/DiffIT.java

```diff
@@ -0,0 +1,60 @@
+package de.retest.recheck.cli.subcommands;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import java.io.File;
+import java.io.IOException;
+
+import org.junit.Before;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.contrib.java.lang.system.SystemOutRule;
+import org.junit.rules.TemporaryFolder;
+
+import de.retest.recheck.cli.util.ReportCreator;
+import picocli.CommandLine;
+
+public class DiffIT {
+
+	private final TemporaryFolder temp = new TemporaryFolder();
```

Comment:
Isn't here `@Rule` missing? Then you can probably also omit the `setUp()` method.